how-to-contribute/development-contribution.md

```diff
@@ -1,125 +0,0 @@
-
```

Comment:
I don't think we should take this out until we have something to replacement. There's a design decisions doc in development, as well as a comparison doc that Dawn, Daniel and Sean are working on.